sklego/model_selection.py

```diff
@@ -643,14 +643,3 @@ def _regroup(self, groups):
         # create a mapper to set every group to the right group_id
         mapper = dict(zip(df["index"], df["group"]))
         return np.vectorize(mapper.get)(groups)
-
-    def _method_is_fitted(self, X):
-        """
-        :param X: Array to use if the method is fitted
-        :return: True if fitted, else False
-        """
-        try:
-            self.cluster_method.predict(X[0:1, :])
-            return True
-        except NotFittedError:
-            return False
```

Comment:
Looks like this method was copy-pasted from `KlusterFoldValidation`, and requires `cluster_method` which is not available for `GroupTimeSeriesSplit`